src/nvim/buffer_defs.h

```diff
@@ -778,7 +778,10 @@ struct file_buffer {
                                  * normally points to this, but some windows
                                  * may use a different synblock_T. */
 
-  signlist_T *b_signlist;       /* list of signs to draw */
+  signlist_T *b_signlist;       // list of signs to draw
+  int b_max_signs_per_line;     // cached maximum number of signs in a line
+  int b_colsigns;               // last calculated maximum number of signs
```

Comment:
let's rename these so that all signs-related fields shared the same `b_sign` prefix.

- `b_max_signs_per_line` => `b_signcols_max`
- `b_colsigns` => `b_signcols`

Reply:
Agreed - fixed.